go.sh

```diff
@@ -20,5 +20,36 @@
 #    files (e.g. signed public releases should match signed intel releases).
 #    "publish" rule. 
 
-echo "Nothing implemented"
+ROOT=/
+INTEL_PATH=lib/firmware/intel
+VERSION=v1.4.3
```

Comment:
I wonder here if we want to put this in master and have this script pull the version from the branch name. This would mean we would need a single script in master (which can be merged with branches) @xiulipan any comments ?

Reply:
@lgirdwood That would be a good idea. We should move this scripts into master code base.
PS: I think the version info is not matter here as we have only one version fw here.